idea/src/main/kotlin/io/paddle/idea/execution/state/PythonScriptCommandLineStateProvider.kt

```diff
@@ -21,6 +21,7 @@ class PythonScriptCommandLineStateProvider : PaddleTaskRunProfileStateProvider<R
             scriptParameters = task.arguments.joinToString(" ")
             sdkHome = module.pythonSdk?.homePath
             isModuleMode = task.isModuleMode
+            workingDirectory = context.rootDir.absolutePath
```

Comment:
Are you sure it should be `rootDir`, and not the `workingDir` of the `task.project`? Is it right if the task is executed for some nested subproject, and not the root project?

Reply:
Not really sure. Probably `workingDir` is more clear and makes more sense, I'll change it